ui/app/AppLayouts/Chat/panels/InlineSelectorPanel.qml

```diff
@@ -286,8 +286,5 @@ Item {
                 onCountChanged: currentIndex = 0
             }
         }
-        onClosed: {
-            suggestionsDialog.destroy();
```

Comment:
This one made it impossible to get inline contact suggestions in the "Create Chat" view